_examples/beh-2015-sequencing.md

```diff
@@ -6,64 +6,64 @@ permalink: /examples/beh-2015-sequencing/
+<img src="../../images/beh-2015-sequencing-approach.jpg" width=70% alt="Conceptual representation of the scenario driven optimal sequencing approach. Three steps are involved: 1) the generation of the portfolio of optimal sequences under various future conditions, 2) sensitivity analysis to evaluate the robustness of the optimal sequences and identify influential critical risk factors, 3) the selection of the optimal sequence for implementation.">
 <div class=imgcredit>Figure 1 <a href="https://doi.org/10.1016/j.envsoft.2015.02.006">Beh et al. 2015</a></div>
 </p>
 <ol>
-<li> Step 1 involves the generation of the portfolio of optimal sequences. As for <a href="../theory/robust-decision-making/"> robust decision making </a> approaches, this step aims at identifying plausible solutions able to perform differently under different future conditions. Three sub-steps are considered:
+<li> Step 1 consists of the generation of the portfolio of optimal sequences. As for <a href="../theory/robust-decision-making/"> robust decision making </a> approaches, this step aims at identifying plausible solutions able to perform under different future conditions. Three sub-steps are involved:
 <ol>
 <li> identifying uncertain variables and their plausible ranges over the planning horizon </li>
-<li> identifying plausible future scenarios combining uncertain variables and asociated values </li>
-<li> generating <a href="../theory/pareto-optimal/">Pareto-optimal </a> sequences for each scenario with respect to pre-defined constraints for the different objective functions. This step involves the use of multi-objective evolutionary algorithms (MOEAs) to generate the front of (near) Pareto-optimal solutions. </li>
+<li> identifying plausible future scenarios with different combinations of values of uncertain variables </li>
+<li> generating <a href="../theory/pareto-optimal/">Pareto-optimal </a> sequences for each scenario with respect to pre-defined constraints for the different objective functions. This step may involve the use of multi-objective evolutionary algorithms (MOEAs). </li>
 </ol>
-<li> Step 2 consists of the evaluation of the sensitivity of the Pareto-optimal sequences identified in step 1 to changes in conditions (values) for different uncertain future variables. This step also involves the identification of influential uncertain variables on the sensitivity analysis results. The identifcation of the nature of influential factors would be deterministic on the selection of adaptation measures. Points to consider are: 
+<li> Step 2 consists of a sensitivity analysis. It evaluates the performance the Pareto-optimal sequences identified in step 1 under a range of future conditions and the relative contribution of uncertain variables to the performance. Points to consider are: 
```

Comment:
This changes the meaning - evaluation of sensitivity to changes vs evaluating performance under a range of conditions. I think the former is correct for this paper?

Reply:
Actually, from the Figure it does look like it's first the latter, then the former?